setup.py

```diff
@@ -11,7 +17,7 @@
     url='https://github.com/m3nu/invoice2data',
     description='Python parser to extract data from pdf invoice',
     license="MIT",
-    long_description=open(path.join(path.dirname(__file__), 'README.rst')).read(),
+    long_description=readme_description(),
```

Comment:
This is probably not even needed any more. The new PyPi version will find README.md/rst/etc anyways.

Reply:
@m3nu ok so I will remove this line from `setup.py` and created method readme_description()